src/main/scala/org/opennetworkinsight/proxy/ProxySchema.scala

```diff
@@ -0,0 +1,39 @@
+package org.opennetworkinsight.proxy
+
+object ProxySchema {
+
+  // fields from the input
+
+  val Date = "p_date"
+  val Time = "p_time"
+  val ClientIP = "clientip"
+  val Host = "host"
+  val ReqMethod = "reqmethod"
+  val UserAgent = "useragent"
+  val ResponseContentType = "resconttype"
```

Comment:
Are you talking about Duration? 
I'm seeing a few fields called "int" with a space between. Is that the correct column name or is it the name+type?